editor/src/messages/tool/tool_messages/select_tool.rs

```diff
@@ -568,6 +562,13 @@ impl Fsm for SelectToolFsmState {
 						);
 					}
 					tool_data.drag_current = mouse_position + closest_move;
+
+					if input.keyboard.get(duplicate as usize) && tool_data.not_duplicated_layers.is_none() {
+						tool_data.start_duplicates(document, responses);
+					} else if !input.keyboard.get(duplicate as usize) && tool_data.not_duplicated_layers.is_some() {
+						tool_data.stop_duplicates(responses);
+					}
+
```

Comment:
@0HyperCube do you foresee any problem with moving this down like I did here? It was needed to fix a message ordering issue with the overlays. I haven't noticed any problems but haven't QA tested it extensively. Thanks.